sdk/azcore/arm/resource_type.go

```diff
@@ -11,19 +11,17 @@ import (
 	"strings"
 )
 
-var (
-	// SubscriptionResourceType is the ResourceType of a subscription
-	SubscriptionResourceType = NewResourceType(builtInResourceNamespace, "subscriptions")
+// SubscriptionResourceType is the ResourceType of a subscription
+var SubscriptionResourceType = NewResourceType(builtInResourceNamespace, "subscriptions")
 
-	// ResourceGroupResourceType is the ResourceType of a resource group
-	ResourceGroupResourceType = NewResourceType(builtInResourceNamespace, "resourceGroups")
+// ResourceGroupResourceType is the ResourceType of a resource group
+var ResourceGroupResourceType = NewResourceType(builtInResourceNamespace, "resourceGroups")
```

Comment:
Conforming to a style rule or just something you wanted to change?

Reply:
Style rule, when `var`s or `const`s are grouped together, they are displayed as a code block.